core/templates/dev/head/services/explorationContextService.js

```diff
@@ -93,6 +93,17 @@ oppia.factory('explorationContextService', [
             'ERROR: explorationContextService should not be used outside the ' +
             'context of an exploration.');
         }
+      },
+
+      // Following variable helps to know whether exploration editor is
+      // in main editing mode or preview mode.
+      isInExplorationEditorMode: function() {
+        if (this.getPageContext() === PAGE_CONTEXT.EDITOR &&
```

Comment:
Ok thanx I will do changes, I just thought returning true or false would be more appropriate.